lib/iam_ssoe_oauth/configuration.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module IamSsoeAuth
```

Comment:
If you haven't you may want to consult with BE Tools on best inflection for this module name.
They've been doing a lot of improving inflections for Zeitwerk compliance.
SSOe already has [an entry](https://github.com/department-of-veterans-affairs/vets-api/blob/10cb3dc3c0be5f3c86503b0c1a7f13abbad3c7ef/config/initializers/inflections.rb#L28), for example.
cc: @erluti, who I know has done a lot of that work.

Reply:
Also just noticed that the path is `iam_ssoe_oauth` and the module name is `IamSsoeAuth` (and not `IamSsoeOauth`